letsencrypt/models/base_config_settings.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    letsencrypt_dns_provider = fields.Selection(
+        [('shell', 'Shell')],
+        string='DNS Provider',
+        help='Set this to Shell if you want an external script to be run.')
+    letsencrypt_shell = fields.Text()
+    letsencrypt_url = fields.Char(string='URL')
```

Comment:
what is this good for?

Reply:
The tests where supposed to set the testing url in order to do their jobs but since we cannot test this (due to the reasons you mention above) I am removing this.